docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -100,6 +100,11 @@ In the AppVeyor CI, we run two different passes:
 
 In each case, tests are executed with appropriate privileges.
 
+Tests that need to be run with sudo **on unix systems** should be additionally marked with 'RequireSudoOnUnix' Pester tag.
```

Comment:
`Unix` should be capitalized.  I think it's worth calling out that `RequireSudoOnUnix` is mutually exclusive to all other tags like `CI`, `Feature` (which are now ignored if `RequireSudoOnUnix` is present) and is treated as `CI`.